<!doctype html>
<html>
  <head>
    <meta charset="utf-8">
    
    <!-- Always force latest IE rendering engine or request Chrome Frame -->
    <meta content="IE=edge,chrome=1" http-equiv="X-UA-Compatible">
    
    <!-- Use title if it's in the page YAML frontmatter -->
    <title>The Middleman</title>
    
    <link href="../stylesheets/all.css" rel="stylesheet" />
  </head>
  
  <body>
    <div class="container genes genes_SLC9A6">
      
<!-- <div class="row"> -->
  <!-- <div class="col-md-6">
    <p><a href="../">back to genes</a></p>
  </div> -->
<!-- </div> -->

<!-- <div class="row "> -->
  <div class="col-md-6">
    <p><a href="../">back to genes</a></p>
    <h1><em>SLC9A6</em></h1>
    <p><b>Gene name:  </b><em>solute carrier family 9, subfamily A (NHE6, cation proton antiporter 6), member 6</em></p>
    <p><b>Chromosome band:  </b><em>Xq26.3</em></p>
    <h3>Tier 3</h3>
    <!-- <p class="lead">14 pathogenic variants in 11 papers</p> -->
    <!-- <ul> -->
      <!-- <li class="list-unstyled"> -->
	<!-- <strong> -->
    <!-- 14 -->
	<!-- </strong> -->
	<!-- Implicated in Intellectual Disability/Developmental Delay (ID) -->
      <!-- </li> -->
      <!-- <li class="list-unstyled"> -->
	<!-- <strong> -->
    <!-- 1 -->
	<!-- </strong> -->
	<!-- Implicated in Autism Spectrum Disorder (ASD) -->
      <!-- </li> -->
      <!-- <li class="list-unstyled"> -->
	<!-- <strong> -->
    <!-- 11 -->
	<!-- </strong> -->
	<!-- Implicated in Epilepsy (EP) -->
      <!-- </li> -->
      <!-- <li class="list-unstyled"> -->
	<!-- <strong> -->
    <!-- 0 -->
	<!-- </strong> -->
	<!-- Implicated in Attention Deficit and Hyperactivity Disorder (ADHD) -->
      <!-- </li> -->
      <!-- <li class="list-unstyled"> -->
	<!-- <strong> -->
    <!-- 0 -->
	<!-- </strong> -->
	<!-- Implicated in Schizopherenia (SCZ) -->
      <!-- </li> -->
      <!-- <li class="list-unstyled"> -->
	<!-- <strong> -->
    <!-- 0 -->
	<!-- </strong> -->
	<!-- Implicated in Bipolar Disorder (BD) -->
      <!-- </li> -->
    <!-- </ul> -->
    <!-- <div class="row"> -->
      <div class="col-md-12" id="barGraph">
      </div>
    <!-- </div> -->

  </div>
  <!--  -->

<!-- </div> -->

<div class="row" id="top-pieChart">
  <div class="col-md-6" id="pieChart">
  </div>
</div>


<p id="par">
  <b>Columns to display:</b><br>
  &nbsp; Note: categories denoted by + may not have values in all cases.
</p>

<div class="col-md-3 checkboxes">
  <p><input type="checkbox" name="disorders" checked="checked" /> Disorder(s)<br>
  <input type="checkbox" name="variant_type" checked="checked" /> Variant Type<br>
  <input type="checkbox" name="inheritance" checked="checked" /> Inheritance<br>
  <input type="checkbox" name="method" checked="checked" /> Method<br>
  <input type="checkbox" name="chromosome" /> Chromosome +<br>
  <input type="checkbox" name="cnv_size" /> CNV Size +<br>
  <input type="checkbox" name="genome_build" /> Genome Build +</p>
</div>
<div class="col-md-9 checkboxes">
  <p><input type="checkbox" name="coding_dna_change" /> Coding DNA Change +<br>
  <input type="checkbox" name="protein_change" /> Protein Change +<br>
  <input type="checkbox" name="position" /> Position +<br>
  <input type="checkbox" name="reference_sequence" /> Reference Sequence +<br>
  <input type="checkbox" name="alternate_sequence" /> Alternate Sequence +<br>
  <input type="checkbox" name="patient_id" /> Patient ID<br>
  <input type="checkbox" name="the_reference" checked="checked" /> Reference</p>
</div>
<div class="row">
  <div class="col-md-12" id="table-container">
    <table class="table" id="maintable">
    <thead>
  <tr>
  <th class="disorders">Disorder(s)</th>
  <th class="variant_type">Variant Type</th>
  <th class="inheritance">Inheritance</th>
  <th class="method">Method</th>
  <th class="chromosome">Chromosome</th>
  <th class="cnv_size">CNV Size</th>
  <th class="genome_build">Genome Build</th>
  <th class="coding_dna_change">Coding DNA Change</th>
  <th class="protein_change">Protein Change</th>
  <th class="position">Position</th>
  <th class="reference_sequence">Reference Sequence</th>
  <th class="alternate_sequence">Alternate Sequence</th>
  <th class="patient_id">Patient ID</th>
  <th class="the_reference">Reference</th>
  </tr>
</thead>





<tbody>
<!--  -->
<tr>
<td class="disorders">ID</td>
<td class="variant_type">Splice site</td>
<td class="inheritance">Maternal</td>
<td class="method">Whole exome sequencing</td>
<td class="chromosome"></td>
<td class="cnv_size"></td>
<td class="genome_build"></td>
<td class="coding_dna_change"></td>
<td class="protein_change"></td>
<td class="position">135095506</td>
<td class="reference_sequence">A</td>
<td class="alternate_sequence">G</td>
<td class="patient_id">261391</td>
<td class="the_reference"><a href="#references">Fitzgerald et al. 2014</a></td>
<!--  -->
<tr>
<td class="disorders">ID, EP</td>
<td class="variant_type">Nonsense</td>
<td class="inheritance">Maternal</td>
<td class="method">Targeted sequencing</td>
<td class="chromosome"></td>
<td class="cnv_size"></td>
<td class="genome_build"></td>
<td class="coding_dna_change"></td>
<td class="protein_change">p.R468X</td>
<td class="position"></td>
<td class="reference_sequence"></td>
<td class="alternate_sequence"></td>
<td class="patient_id">B III-1</td>
<td class="the_reference"><a href="#references">Gilfillan et al. 2008</a></td>
<!--  -->
<tr>
<td class="disorders">ID, EP</td>
<td class="variant_type">Splice site</td>
<td class="inheritance">Maternal</td>
<td class="method">Targeted sequencing</td>
<td class="chromosome"></td>
<td class="cnv_size"></td>
<td class="genome_build"></td>
<td class="coding_dna_change"></td>
<td class="protein_change">p.V144_R169 del</td>
<td class="position"></td>
<td class="reference_sequence"></td>
<td class="alternate_sequence"></td>
<td class="patient_id">C II-1</td>
<td class="the_reference"><a href="#references">Gilfillan et al. 2008</a></td>
<!--  -->
<tr>
<td class="disorders">ID, EP</td>
<td class="variant_type">Frameshift</td>
<td class="inheritance">Unknown</td>
<td class="method">Targeted sequencing</td>
<td class="chromosome"></td>
<td class="cnv_size"></td>
<td class="genome_build"></td>
<td class="coding_dna_change"></td>
<td class="protein_change">p.H171fs</td>
<td class="position"></td>
<td class="reference_sequence"></td>
<td class="alternate_sequence"></td>
<td class="patient_id">D 1</td>
<td class="the_reference"><a href="#references">Gilfillan et al. 2008</a></td>
<!--  -->
<tr>
<td class="disorders">ID, ASD, EP</td>
<td class="variant_type">Nonsense</td>
<td class="inheritance">Unknown</td>
<td class="method">Targeted sequencing</td>
<td class="chromosome"></td>
<td class="cnv_size"></td>
<td class="genome_build"></td>
<td class="coding_dna_change"></td>
<td class="protein_change">p.Gln306X</td>
<td class="position"></td>
<td class="reference_sequence"></td>
<td class="alternate_sequence"></td>
<td class="patient_id">NA</td>
<td class="the_reference"><a href="#references">Mignot et al. 2013</a></td>
<!--  -->
<tr>
<td class="disorders">ID</td>
<td class="variant_type">Splice site</td>
<td class="inheritance">Maternal</td>
<td class="method">Targeted sequencing</td>
<td class="chromosome"></td>
<td class="cnv_size"></td>
<td class="genome_build"></td>
<td class="coding_dna_change">c.526-9_526-5del</td>
<td class="protein_change">p.?; splice disrupted</td>
<td class="position"></td>
<td class="reference_sequence"></td>
<td class="alternate_sequence"></td>
<td class="patient_id">APN-43</td>
<td class="the_reference"><a href="#references">Redin et al. 2014</a></td>
<!--  -->
<tr>
<td class="disorders">ID, EP</td>
<td class="variant_type">Frameshift</td>
<td class="inheritance">Maternal</td>
<td class="method">Targeted sequencing</td>
<td class="chromosome"></td>
<td class="cnv_size"></td>
<td class="genome_build"></td>
<td class="coding_dna_change">c.1464_145insT</td>
<td class="protein_change">p.Thr489TyrfsX23</td>
<td class="position"></td>
<td class="reference_sequence"></td>
<td class="alternate_sequence"></td>
<td class="patient_id">Family 1</td>
<td class="the_reference"><a href="#references">Riess et al. 2013</a></td>
<!--  -->
<tr>
<td class="disorders">ID, EP</td>
<td class="variant_type">Nonsense</td>
<td class="inheritance">Maternal</td>
<td class="method">Targeted sequencing</td>
<td class="chromosome"></td>
<td class="cnv_size"></td>
<td class="genome_build"></td>
<td class="coding_dna_change">c.1402C>T</td>
<td class="protein_change">p.R468X</td>
<td class="position"></td>
<td class="reference_sequence"></td>
<td class="alternate_sequence"></td>
<td class="patient_id">K8310 III-9</td>
<td class="the_reference"><a href="#references">Schroer et al. 2010</a></td>
<!--  -->
<tr>
<td class="disorders">ID, EP</td>
<td class="variant_type">Nonsense</td>
<td class="inheritance">Maternal</td>
<td class="method">Targeted sequencing</td>
<td class="chromosome"></td>
<td class="cnv_size"></td>
<td class="genome_build"></td>
<td class="coding_dna_change">c.1219C>T</td>
<td class="protein_change">p.Q407X</td>
<td class="position"></td>
<td class="reference_sequence"></td>
<td class="alternate_sequence"></td>
<td class="patient_id">K9524</td>
<td class="the_reference"><a href="#references">Schroer et al. 2010</a></td>
<!--  -->
<tr>
<td class="disorders">ID, EP</td>
<td class="variant_type">Nonsense</td>
<td class="inheritance">Maternal</td>
<td class="method">Whole exome sequencing</td>
<td class="chromosome"></td>
<td class="cnv_size"></td>
<td class="genome_build"></td>
<td class="coding_dna_change">c.1639G>T</td>
<td class="protein_change">p.Glu547*</td>
<td class="position"></td>
<td class="reference_sequence"></td>
<td class="alternate_sequence"></td>
<td class="patient_id">W08-0135</td>
<td class="the_reference"><a href="#references">Schuurs-Hoeijmakers et al. 2013</a></td>
<!--  -->
<tr>
<td class="disorders">ID, EP</td>
<td class="variant_type">Frameshift</td>
<td class="inheritance">Maternal</td>
<td class="method">Targeted sequencing</td>
<td class="chromosome"></td>
<td class="cnv_size"></td>
<td class="genome_build"></td>
<td class="coding_dna_change">c.441delG</td>
<td class="protein_change">p.S147fs</td>
<td class="position"></td>
<td class="reference_sequence"></td>
<td class="alternate_sequence"></td>
<td class="patient_id">P1</td>
<td class="the_reference"><a href="#references">Takahashi et al. 2011</a></td>
<!--  -->
<tr>
<td class="disorders">ID</td>
<td class="variant_type">Nonsense</td>
<td class="inheritance"><em>De novo</em></td>
<td class="method">Targeted sequencing</td>
<td class="chromosome"></td>
<td class="cnv_size"></td>
<td class="genome_build"></td>
<td class="coding_dna_change">c.2078T>G</td>
<td class="protein_change">p.Leu693*</td>
<td class="position"></td>
<td class="reference_sequence"></td>
<td class="alternate_sequence"></td>
<td class="patient_id">11</td>
<td class="the_reference"><a href="#references">Tan et al. 2015</a></td>
<!--  -->
<tr>
<td class="disorders">ID, EP</td>
<td class="variant_type">Deletion</td>
<td class="inheritance">Unknown</td>
<td class="method">Targeted sequencing</td>
<td class="chromosome">chrX</td>
<td class="cnv_size">335 bp</td>
<td class="genome_build">hg19</td>
<td class="coding_dna_change"></td>
<td class="protein_change"></td>
<td class="position"></td>
<td class="reference_sequence"></td>
<td class="alternate_sequence"></td>
<td class="patient_id">16</td>
<td class="the_reference"><a href="#references">Tzschach et al. 2015</a></td>
<!--  -->
<tr>
<td class="disorders">ID, EP</td>
<td class="variant_type">Splice site</td>
<td class="inheritance">Maternal</td>
<td class="method">Targeted sequencing</td>
<td class="chromosome"></td>
<td class="cnv_size"></td>
<td class="genome_build"></td>
<td class="coding_dna_change"></td>
<td class="protein_change">IVS10-1G>A</td>
<td class="position"></td>
<td class="reference_sequence"></td>
<td class="alternate_sequence"></td>
<td class="patient_id">NA</td>
<td class="the_reference"><a href="#references">Zanni et al. 2014</a></td>
<!--  -->
</tbody>
    </table>
    <div id="bottom_anchor"></div>
  </div>
</div>
<!--  -->

<div class="row" id="citations">
  <h4>References</h4>
  <div class="col-md-12">
    <p><a name="references"></a>
      Deciphering Developmental Disorders Study
      ,
      <a href="http://www.ncbi.nlm.nih.gov/pubmed/25533962">Large-scale discovery of novel genetic causes of developmental disorders.</a>
      Nature
      2015
    </p>
  </div>
  <div class="col-md-12">
    <p><a name="references"></a>
      Gilfillan GD
      et. al.,
      <a href="http://www.ncbi.nlm.nih.gov/pubmed/18342287">SLC9A6 mutations cause X-linked mental retardation, microcephaly, epilepsy, and ataxia, a phenotype mimicking Angelman syndrome.</a>
      Am J Hum Genet
      2008
    </p>
  </div>
  <div class="col-md-12">
    <p><a name="references"></a>
      Mignot C
      et. al.,
      <a href="http://www.ncbi.nlm.nih.gov/pubmed/22541666">Novel mutation in SLC9A6 gene in a patient with Christianson syndrome and retinitis pigmentosum.</a>
      Brain Dev
      2013
    </p>
  </div>
  <div class="col-md-12">
    <p><a name="references"></a>
      Redin C
      et. al.,
      <a href="http://www.ncbi.nlm.nih.gov/pubmed/25167861">Efficient strategy for the molecular diagnosis of intellectual disability using targeted high-throughput sequencing.</a>
      J Med Genet
      2014
    </p>
  </div>
  <div class="col-md-12">
    <p><a name="references"></a>
      Riess A
      et. al.,
      <a href="http://www.ncbi.nlm.nih.gov/pubmed/22931061">Novel SLC9A6 mutations in two families with Christianson syndrome.</a>
      Clin Genet
      2013
    </p>
  </div>
  <div class="col-md-12">
    <p><a name="references"></a>
      Schroer RJ
      et. al.,
      <a href="http://www.ncbi.nlm.nih.gov/pubmed/20949524">Natural history of Christianson syndrome.</a>
      Am J Med Genet A
      2010
    </p>
  </div>
  <div class="col-md-12">
    <p><a name="references"></a>
      Schuurs-Hoeijmakers JH
      et. al.,
      <a href="http://www.ncbi.nlm.nih.gov/pubmed/24123876">Identification of pathogenic gene variants in small families with intellectually disabled siblings by exome sequencing.</a>
      J Med Genet
      2013
    </p>
  </div>
  <div class="col-md-12">
    <p><a name="references"></a>
      Takahashi Y
      et. al.,
      <a href="http://www.ncbi.nlm.nih.gov/pubmed/21812100">A loss-of-function mutation in the SLC9A6 gene causes X-linked mental retardation resembling Angelman syndrome.</a>
      Am J Med Genet B Neuropsychiatr Genet
      2011
    </p>
  </div>
  <div class="col-md-12">
    <p><a name="references"></a>
      Tan CA
      et. al.,
      <a href="http://www.ncbi.nlm.nih.gov/pubmed/25693842">Characterization of patients referred for non-specific intellectual disability testing: the importance of autosomal genes for diagnosis.</a>
      Clin Genet
      2015
    </p>
  </div>
  <div class="col-md-12">
    <p><a name="references"></a>
      Tzschach A
      et. al.,
      <a href="http://www.ncbi.nlm.nih.gov/pubmed/25649377">Next-generation sequencing in X-linked intellectual disability.</a>
      Eur J Hum Genet
      2015
    </p>
  </div>
  <div class="col-md-12">
    <p><a name="references"></a>
      Zanni G
      et. al.,
      <a href="http://www.ncbi.nlm.nih.gov/pubmed/24630051">A novel mutation in the endosomal Na+/H+ exchanger NHE6 (SLC9A6) causes Christianson syndrome with electrical status epilepticus during slow-wave sleep (ESES).</a>
      Epilepsy Res
      2014
    </p>
  </div>
</div>

<div class="source_data" data-unknown="3" data-denovo="1" data-inherited="10"></div>

<div class="implicated_data" data-ID="14" data-ASD="1" data-EP="11" data-ADHD="0" data-SCZ="0" data-BD="0"></div>

    </div>
  </body>
  <script src="../javascripts/all.js"></script><script src="../javascripts/bootstrap.js"></script>
</html>
